fmralign/tests/test_hungarian_alignment.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
```

Comment:
I propose to create no mroe than on test file per module: this keeps the code and test base much more readable for people. So gather the tests into a test_alignment.py file.

Reply:
It's done